component-maps.yml

```diff
@@ -77,9 +80,11 @@ git:
 
   mender:
     docker_image:
-    - mender-client-docker
```

Comment:
`mender` should retain `mender-client-docker`, it's mender-connect where it can be removed.

Reply:
https://github.com/mendersoftware/integration/pull/1835/files/170f2db294ed75122e808c371af80300e93b3fa4#diff-37a5a19da4dabae31f06c276f64ff61b5d12bc2ed734bd377000e99d69142d38R85

:)